### ISSUE TITLE
`key_press_enables_dpms` doesn't work.

### ISSUE THREAD

**Issue by HanoJing** [hyprwm/Hyprland#1769]
the new option `key_press_enables_dpms`  doesn't work, at least for me.

- Steps to reproduce: add `key_press_enables_dpms = true` to `misc` in the `hyprland.conf`. Disable DPMS
- Expected outcome: DPMS turns on again, when any key on the keyboard is pressed.
- Noted outcome: DPMS does not turn on again. when any key on the keyboard is pressed.

Notes:
- `mouse_move_enables_dpms` works fine for me.
- I use a 65% USB keyboard.

- I use this script to disable DPMS and lock the session:
```
#!/bin/zsh

hyprctl dispatch dpms off;
swaylock -c 000000;
hyprctl dispatch dpms on;
```

**Comment by vaxerski** [comment]
[patch.txt](https://github.com/hyprwm/Hyprland/files/10952563/patch.txt)

try this

**Comment by HanoJing** [comment]
The patch sadly didn't fix it.

**Comment by vaxerski** [comment]
oh god sorry I think I was actually braindead or something, fixed in 3e261b1fa78d13daeda8bede7df3f2c6af032c46

**Comment by HanoJing** [comment]
No worries! You are doing great work regardless and I admire how active you are on Hyprland!
